packages/react-vapor/src/components/numericInput/NumericInputConnected.tsx

```diff
@@ -86,7 +85,20 @@ export class NumericInputConnected extends React.PureComponent<NumericInputProps
                     </Button>
                     <div className="flex flex-column">
                         <input
-                            {..._.omit(this.props, keys<NumericInputProps>())}
+                            {..._.omit(this.props, [
+                                'hasError',
+                                'id',
+                                'initialValue',
+                                'invalidMessage',
+                                'max',
+                                'maxLength',
+                                'min',
+                                'mount',
+                                'setValue',
+                                'step',
+                                'unmount',
+                                'value',
+                            ])}

```

Comment:
For consistency, should be extract this array to `inputPropsToOmit` or something?

Reply:
Done